pkg/pxc/backup/scheduled.go

```diff
@@ -24,7 +24,7 @@ func (bcp *Backup) Scheduled(spec *api.PXCScheduledBackupSchedule, strg *api.Bac
 	labels["cluster"] = bcp.cluster
 	labels["schedule"] = genScheduleLabel(spec.Schedule)
 
-	cjts, err := bcp.scheduledJob(spec, strg, labels, operatorPod.Spec.ServiceAccountName)
+	cjts, err := bcp.scheduledJob(spec, strg, labels, operatorPod.Spec.ServiceAccountName, operatorPod.Spec.RuntimeClassName)
```

Comment:
NodeSelector and RuntimeClassName are very connected things.
so, both items should be set on the same storage level (not operatorPod)
```suggestion
	cjts, err := bcp.scheduledJob(spec, strg, labels, operatorPod.Spec.ServiceAccountName)
```